db/migrate/20150112083118_create_atmosphere_action_logs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAtmosphereActionLogs < ActiveRecord::Migration
+  def change
+    create_table :atmosphere_action_logs do |t|
+      t.string :message
+      t.string :log_level
+      t.references :action, index: true
+      t.timestamps
```

Comment:
Action log doesn't need `updated_at`; suggest changing this to:
`t.datetime :created_at`